frontend/packages/console-shared/src/components/modals/DeleteResourceModal.tsx

```diff
@@ -78,12 +80,12 @@ class DeleteResourceModal extends PromiseComponent<
   DeleteResourceModalState
 > {
   private handleSubmit = (values, actions) => {
-    actions.setSubmitting(true);
+    actions.setStatus({ isSubmitting: true });
```

Comment:
This is just a hack to make this work and not a good way to set submitting status. Formik handles submitting status for you automatically if the submit handler is async and returns a promise. You can read more about why this doesn't work right now in this [code comment](https://github.com/formium/formik/blob/master/packages/formik/src/Formik.tsx#L748)